tools/cli/domain_commands.go

```diff
@@ -261,18 +258,14 @@ func (d *domainCLIImpl) UpdateDomain(c *cli.Context) {
 	updateRequest.SecurityToken = securityToken
 	_, err := d.updateDomain(ctx, updateRequest)
 	if err != nil {
-		if _, ok := err.(*types.EntityNotExistsError); !ok {
-			ErrorAndExit("Operation UpdateDomain failed.", err)
-		} else {
-			ErrorAndExit(fmt.Sprintf("Domain %s does not exist.", domainName), err)
-		}
-	} else {
-		fmt.Printf("Domain %s successfully updated.\n", domainName)
```

Comment:
seems like a moderate loss, unless something makes "not exist" not possible / that was dead code

Reply:
make sense. I'll revert it. Originally I was just relying on error details.